### ISSUE TITLE
Crossref to also crossref file

### ISSUE THREAD

**Issue by ilippert** [JabRef/jabref#7731]
I have an entry A, say an edited volume, and have linked to it a file AF.

Entry B crossreferences A.

The entry table successfully uses some data of A to present data for B.
I am frustrated because the link to AF is not shown for B in the entry table.

Proposal. IFF B does not have own files attached, offer AF in the entry table.

**Comment by calixtus** [comment]
Maybe a bindings problem with the reference to the crossrefd bibentry...?

**Comment by k3KAW8Pnf7mkmdSMPHz27** [comment]
It is possibly solved if #7730 is fixed (as long as we are talking about the entry-editor and not the maintable). It should be mitigated by #7754. I am not sure if all the buttons/menus will work as expected for files after #7730.

**Comment by calixtus** [comment]
Monday I hot word that maybe some students from a computer science course are looking into this issue the next days.

**Comment by k3KAW8Pnf7mkmdSMPHz27** [comment]
Ok. I don't know what the simplest solution is in that case.

I haven't seen any new PR/commits from @SuXiChangZhen so I am assuming it is ok that I finish the currently open PR and someone else can continue with a complete solution (or a complete solution only for the file field).
I think the last of the suggestions in https://github.com/JabRef/jabref/pull/7754#discussion_r641697015 is the most likely to work but I am not sure if that is a "good first issue" anymore.

I can expand on the comments there, but I am going to be away for 10 days starting on Friday and I am not sure if I'll have cellphone coverage.

**Comment by k3KAW8Pnf7mkmdSMPHz27** [comment]
Ok, turns out that I was wrong. #7754 has probably nothing to do with this one since it only is relevant for the main table, not the entry editor. Terribly sorry for the confusion!

**Comment by RhodevdDool** [comment]
Hi, is anyone currently working on this? If not, I'd like to give it a try.
Apart from the devdocs and the above information, is there anything specific I should know about this?

**Comment by calixtus** [comment]
Hi @RhodevdDool, thank you for your interest in JabRef programming. We always appreciate if people want to contribute. About this issue: Currently there is a student group from china (?) working on this. Their teacher told me, that they should be done on the 8. of august. If there is no PR then present, this issue should be available again.

**Comment by RhodevdDool** [comment]
@calixtus Thank you! I used JabRef for my thesis, and am excited to maybe contribute in turn. I'll check it out in a week.

**Comment by calixtus** [comment]
😍 This is exactly how most of us started with JabRef programming! Either as a distraction from the thesis or - just like you - after we were done. But don't forget: Working on JabRef should be fun!

**Comment by ilippert** [comment]
dear community, 
just asking for clarification - is anybody of you still working on this? It would be so great to see progress on this issue.

**Comment by HighCrit** [comment]
If no one else is working on this. Could I be assigned to this issue? I wish to work on it as part of a uni course.

**Comment by Toro520** [comment]
Hi there, 
I'm new here. I'm interested about it and wanna combined with my knowledge and practice together to solve it.  Could I be able to assigned to this issue and have a try? Thanks again.

**Comment by ThiloteE** [comment]
@Toro520 Sure, go ahead, looks like HighCrit abandoned the project.
@HighCrit Thank you for unassigning yourself.

**Comment by koppor** [comment]
This issue somehow refs https://github.com/JabRef/jabref/issues/8128 - even though this issue is talking about a single file and "virtually" presenting it at another place; whereas #8128 also talks about a single file, but different positions in that file (which is currently unclear how to configure).

**Comment by Toro520** [comment]
Hey there,
This is really an awesome application. Had a glance of the code, the skeleton is really complex. I'm considering this issue #7731, so if I wanna fix it, where could I start it, and which package and class should I to edit and have a improve? Thanks guys.

**Comment by ThiloteE** [comment]
@Toro520 have a look at https://github.com/JabRef/jabref/pull/9717. HighCrit already contributed some code, so maybe you can continue from there.

![image](https://github.com/JabRef/jabref/assets/73715071/3d903e89-28b2-4863-817d-6e59a5aeb750)

**Comment by Toro520** [comment]
@ThiloteE Thank you so much.  :)
